lib/kubernetes-deploy/kubernetes_resource/pod.rb

```diff
@@ -120,5 +140,69 @@ def display_logs
 
       @already_displayed = true
     end
+
+    class Container
```

Comment:
Should we add some basic unit tests for this class? Or do you consider it very internal?